traitsui/qt4/tests/test_ui_panel.py

```diff
@@ -110,3 +161,53 @@ def test_subpanel_no_toolbar_nor_button_in_widget(self):
 
             # No button
             self.assertIsNone(ui.control.findChild(QtGui.QPushButton))
+
+
+@requires_toolkit([ToolkitName.qt])
+class TestPanelLayout(unittest.TestCase):
+
+    def test_scrollable_group_typical(self):
+        from pyface.qt import QtGui
+
+        example = ScrollableGroupExample()
+
+        ui = example.edit_traits(view=scrollable_group_view)
+        mainwindow = ui.control.layout().itemAt(0).widget()
+        scroll_area = mainwindow.centralWidget()
+        self.assertIsInstance(scroll_area, QtGui.QScrollArea)
+        content = scroll_area.widget()
+        self.assertIsInstance(content, QtGui.QWidget)
+
+    def test_scrollable_group_box(self):
+        from pyface.qt import QtGui
+
+        example = ScrollableGroupExample()
+
+        ui = example.edit_traits(view=scrollable_group_box_view)
+        mainwindow = ui.control.layout().itemAt(0).widget()
+        scroll_area = mainwindow.centralWidget()
+        self.assertIsInstance(scroll_area, QtGui.QScrollArea)
+        group_box = scroll_area.widget()
+        self.assertIsInstance(group_box, QtGui.QGroupBox)
+
+    def test_scrollable_labelled_group(self):
+        from pyface.qt import QtGui
+
+        example = ScrollableGroupExample()
+
+        ui = example.edit_traits(view=scrollable_labelled_group_view)
+        mainwindow = ui.control.layout().itemAt(0).widget()
+        scroll_area = mainwindow.centralWidget()
+        self.assertIsInstance(scroll_area, QtGui.QScrollArea)
+        group_box = scroll_area.widget()
+        self.assertIsInstance(group_box, QtGui.QWidget)
```

Comment:
The returned widget is not actually a `QGroupBox` so the variable name is a bit misleading (as I was thinking I could do `assertIsInstance(group_box, QtGui.QGroupBox)` instead, but I can't.

Not sure if this is an acceptable amount of implementation details being used to check whether the label is within the scrollable area:
```
        label = scroll_area.findChild(QtGui.QLabel)
        self.assertIn("Scrollable View", label.text())
```

(I remember segfault from ModalDialogTester checking for `text()`. In the context of `ModelDialogTester`, the GUI event loop is running and such query 
 is made from when the event is being processed... Here the GUI event loop is not running and making these searches from the main thread seems safer than the setup we have in ModelDialogTester.)

Reply:
I don't think I want to test for where the label occurs - it can't be on the outside because then we wouldn't get a `QScrollArea` where we would expect it.  Testing the inserted label is really a test for the group `label` trait, not the `scrollable` trait, and so getting beyond the scope of this PR.